src/async_fuse/memfs/cache/mock.rs

```diff
@@ -0,0 +1,163 @@
+//! Mock storages for test and local nodes.
+
+use std::collections::{HashMap, HashSet};
+
+use async_trait::async_trait;
+use parking_lot::Mutex;
+
+use crate::async_fuse::fuse::protocol::INum;
+
+use super::{Block, IoBlock, Storage};
+
+/// A "persistent" storage layer in memory.
+#[derive(Default)]
```

Comment:
If these data structures are used in test only. Please mark it with `#[test]`.

Reply:
I'm not sure if it is only used in test, we may use it as a backend when running local node, until the solution of backend interface is stable.